indico/web/client/js/custom_elements/ind_select.scss

```diff
@@ -0,0 +1,192 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'design_system';
+@use 'partials/icons';
+@use 'partials/sui_debt';
+
+ind-select {
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  padding: 0;
+
+  cursor: pointer;
+
+  .caption {
+    @include sui_debt.input();
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    @include sui_debt.field_override() {
+      padding-right: 2.5em;
+    }
+
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  &:focus .caption {
```

Comment:
I think I fixed this one. It should've been `:focus-visible` instead of `:focus`.

Reply:
I'll look into why the other field receives focus, though. Cause it shouldn't. Maybe it's a Firefox bug or something.

Reply:
I don't see anything obvious in the code that would cause such behavior, so I'd say it's definitely a Firefox bug. Chrome doesn't have this issue. It doesn't affect keyboard users, and we took care of the visual glitch for mouse users, so I'm inclined to say it's good enough as it is.

Reply:
works all fine now :)